Resources/Prototypes/Entities/Mobs/Player/clone.yml

```
@@ -11,6 +11,7 @@
   # general
   - DetailExaminable
   - Dna
+  - Feeble # Moffstation - Add feeble trait
```

```suggestion
  - DisarmProne # Moffstation - Add feeble trait
```
This should fix the testfail.
This list wants the _component_ which the trait adds, not the trait.

---

Ahh, that makes sense, whoops!